test/extensions/transport_sockets/tls/ssl_socket_test.cc

```diff
@@ -779,26 +786,49 @@ void testUtilV2(const TestUtilOptionsV2& options) {
   }
 }
 
-// Configure the listener with unittest{cert,key}.pem and ca_cert.pem.
-// Configure the client with expired_san_uri_{cert,key}.pem
+void updateFilterChain(
+    const envoy::extensions::transport_sockets::tls::v3::DownstreamTlsContext& tls_context,
+    envoy::config::listener::v3::FilterChain* filter_chain) {
```

Comment:
Thanks for pointing this out. I am just wondering since it is already pointer on the caller site. (because `FilterChain* filter_chain = listener.add_filter_chains();` ), do we still need to use reference and deference the pointer here? 
If you or envoy style guide feels strong on this, please let me know and I am happy to change it.

Reply:
Yep, still preferred to pass-by-reference when non-nullable, see https://github.com/envoyproxy/envoy/blob/main/STYLE.md#deviations-from-google-c-style-guidelines.

Reply:
Yep. Thanks! Done